docs/connecting-to-aws.md

```diff
@@ -1,30 +1,43 @@
-  * Profiles with an access key and a secret key ([Named Profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html))
-  * Profiles that [assume a role](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html)
-  * Profiles that [assume a role](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html) and [use multifactor authentication](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html#cli-configure-role-mfa) (MFA)
-    * When connecting with credentials defined to assume a role and use MFA, the Toolkit prompts for an MFA token.
+-   Credential profiles defined in shared credentials files (see [Configuration and Credential Files](https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html)):
+    -   Profiles with an access key and a secret key ([Named Profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html))
+    -   Profiles that [assume a role](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html)
+    -   Profiles that [assume a role](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html) and [use multifactor authentication](https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html#cli-configure-role-mfa) (MFA)
+        -   When connecting with credentials defined to assume a role and use MFA, the Toolkit prompts for an MFA token.
+    -   profiles that use an [external credential process](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/external-credential-process.html)
 
 ## Adding Credentials
 
 Additional credentials can be defined in the shared AWS credentials file. In the VS Code Command Palette, select **AWS: Create Credentials Profile**. The Toolkit does the following:
 
-* If the shared AWS credentials file is not found, the Toolkit prompts for a profile name, an access key ID, and the corresponding secret access key. This information is then used to create an initial credentials file.
-* If the shared AWS credentials file is found, it is opened in VS Code for editing, and the Toolkit provides information about how to change the file.
+-   If the shared AWS credentials file is not found, the Toolkit prompts for a profile name, an access key ID, and the corresponding secret access key. This information is then used to create an initial credentials file.
+-   If the shared AWS credentials file is found, it is opened in VS Code for editing, and the Toolkit provides information about how to change the file.
+
+## Working with Regions in other Partitions
+
+Partitions influence which regions are available for an account to operate in. Many regions commonly used belong to the 'aws' partition, such as 'us-east-1' (N. Virginia). Some regions such as Beijing reside in [other partitions](https://docs.aws.amazon.com/general/latest/gr/rande.html#learn-more).
```

Comment:
awkward wording: "Many regions commonly used belong...". Could instead say "popular" or "frequently-used".

And might as well change the single-quotes to double quotes. Single-quotes in prose are quite informal. 

```suggestion
Partitions influence which regions are available for an account to operate in. Many popular regions belong to the "aws" partition, such as "us-east-1" (N. Virginia). Some regions such as Beijing reside in [other partitions](https://docs.aws.amazon.com/general/latest/gr/rande.html#learn-more).
```